node/src/exchange.rs

```diff
@@ -231,6 +233,7 @@ where
                 return Poll::Ready(ev);
             }
         }
+        let _ = self.server_handler.poll(cx, &mut self.req_resp);
```

Comment:
Based on the [`poll` convention](https://github.com/libp2p/rust-libp2p/blob/master/docs/coding-guidelines.md#conventions-for-poll-implementations), the body of this method should be like this:

```rust
        loop {
            if let Poll::Ready(ev) = self.req_resp.poll(cx, params) {
                if let Some(ev) = self.on_to_swarm(ev) {
                    return Poll::Ready(ev);
                }

                continue;
            }

            if let Poll::Ready(()) = self.server_handler.poll(cx, &mut self.req_resp) {
                continue;
            }

            return Poll::Pending;
        }
```

Reply:
Thanks for the link, havent found that info yet  👍 changed according to guidelines.